README.md

```diff
@@ -6,6 +6,36 @@ The primary purpose of this software is to identify quotes of Karl Marx's _Manif
 der Kommunistischen Partei_ and the first volume of _Das Kapital_ within articles
 published in _Die Neue Zeit_ between 1891 and 1918.
 
+## Basic Usage
+
+### Installation
+
```

Comment:
Let's be explicit that we expect the package to be used with uv and give them a link.    

```suggestion
Documentation assumes the use of `uv` for installing python and python packages and running python scripts. The first time you follow them, you should Install `uv` for per [uv installation documentation](https://docs.astral.sh/uv/getting-started/installation/). This only needs to be done once.
```